mailpoet/assets/js/src/subscribers/importExport/import/step_clean_list.tsx

```diff
@@ -3,7 +3,11 @@ import { CleanList } from './clean_list';
 
 function StepCleanList({ history }: RouteComponentProps): JSX.Element {
   return (
-    <CleanList onProceed={(): void => history.push('step_method_selection')} />
+    <CleanList
+      onProceed={(): void => {
+        history.push('step_method_selection');
+      }}
+    />
```

Comment:
Seems like the `void` was misplaced here. The following should work:

```ts
<CleanList onProceed={() => void history.push('step_method_selection')} />
```

But your solution is also 👍